x-pack/plugins/enterprise_search/public/applications/shared/role_mapping/role_mappings_table.test.tsx

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+
+import { shallow } from 'enzyme';
+
+import { EuiFieldSearch, EuiTableRow, EuiTextColor } from '@elastic/eui';
+
+import { wsRoleMapping, asRoleMapping } from './__mocks__/roles';
+
+import { ALL_LABEL } from './constants';
+
+import { RoleMappingsTable } from './role_mappings_table';
+
+describe('RoleMappingsTable', () => {
+  const getRoleMappingPath = jest.fn();
+  const roleMappings = [wsRoleMapping] as any;
+
+  const props = {
+    accessItemKey: 'groups' as 'groups' | 'engines',
```

Comment:
It's super annoying that we have to do this in TS.

You might consider pulling this into a common type or enum on `/shared/role_mapping/types.ts`

I realize this is how the code was origianlly in ent-search, so feel free to skip this feedback.